schema/JF_TRAN.json

```diff
@@ -284,7 +284,8 @@
       "description": "",
       "type": "number",
       "minimum": 0,
-      "maximum": 999999999999,
+      "maximum": 999999999.99,
+			"pattern": "^[0-9]{1,9}(?:[.][0-9]{2}?)?$",
```

Comment:
I don't think this pattern is supposed to be here

Reply:
Gone